pkg/controller/controller_broker.go

```diff
@@ -408,6 +409,9 @@ func (c *controller) reconcileClusterServiceBroker(broker *v1beta1.ClusterServic
 
 		c.recorder.Event(broker, corev1.EventTypeNormal, successFetchedCatalogReason, successFetchedCatalogMessage)
 
+		metrics.BrokerServiceClassCount.WithLabelValues(broker.Name).Set(float64(len(payloadServiceClasses)))
```

Comment:
Should we also delete the Gauge for the broker when the ClusterServiceBroker is deleted?